addons/website/models/ir_module_module.py

```diff
@@ -416,6 +416,21 @@ def update_theme_images(self):
                         'res_id': theme.id,
                     })
 
+    def get_themes_domain(self):
+        """Returns the 'ir.module.module' search domain matching all available themes."""
+        def get_id(view_id):
+            view = self.env.ref(view_id, False)
```

Comment:
it's not a view/view_id

Reply:
xmlid_to_res_id do the same no?

Reply:
Yes but I think what odoo is saying is the parameter is called `view_id` but the ids passed in have nothing to do with views they're module category xids.

Reply:
yes, it was a second comment unrelated, get_id do no more than the xmlid_to_res_id function from orm.